_targets.R

```diff
@@ -22,13 +24,15 @@ list(
 
   # TODO: PULL SITE DATA HERE
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states)%>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb)),
     tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
     # Insert step for tallying data here
-    tar_target(count, tally(nwis_data)),
+    tar_target(tally, tally_site_obs(nwis_data)),
     # Insert step for plotting data here
-    tar_target(fig, plot(count))
+    tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter)),
```

Comment:
Small change -- this needs to be a file target, since `plot_site_data()` [generates a file and returns a file path](https://github.com/amsnyder/ds-pipelines-targets-3/blob/appliers/3_visualize/src/plot_site_data.R#L11-L12)